.travis.yml

```diff
@@ -51,17 +51,17 @@ matrix:
     - php: 7.1
       env:
         - DEPS=latest
-    - php: nightly
+    - php: 7.2
       env:
         - DEPS=lowest
-    - php: nightly
+    - php: 7.2
       env:
         - DEPS=locked
-    - php: nightly
+    - php: 7.2
       env:
         - DEPS=latest
   allow_failures:
-    - php: nightly
+    - php: 7.2
```

Comment:
Not sure if I am right, but somewhere, someone secretly decided to not allow failures on PHP 7.2 any more ;-)
@froschdesign Is it right? :smile:

Reply:
@samsonasik and @webimpress 
November 30, 2017 is around the corner therefore it make no sense! :smiley:

Reply:
@webimpress @froschdesign I've removed allow_failures->php 7.2, do you think we need to keep the "nightly"?

Reply:
@samsonasik I wouldn't do that as long as we don't have PHP 7.3-alpha yet.